### PR TITLE
update cmd processors: change print func to avoid $ at the end of print line

### PR DESCRIPTION
**Pull request by zhangjieke**
fmt.Fprint(os.Stdout, out) -> fmt.Fprintf(os.Stdout, "%s\n", out)

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ var {{ .Camel }}Cmd = &cobra.Command{
 			return err
 		}
 
-		_, err = fmt.Fprint(os.Stdout, out)
+		_, err = fmt.Fprintf(os.Stdout, "%s\n", out)
```
Comment (abhimanyu003) [inline cmd/generate.go:169]: Can you please explain the use case of new line more. 
Thanks

**Comment by abhimanyu003** [comment]
Hello,
Many thanks for you PR, can you please explain the use-case of newline a bit more?
Also thanks for new processor.

**Comment by abhimanyu003** [comment]
Hello,
Many thanks for your PR, really appreciate your time and poking around with codes.
 Can you please explain the use-case of newline a bit more?
Also thanks for new processor.

File: cmd/generate.go (hunk 1 above)
Comment (zhangjieke) [reply cmd/generate.go:169]: Hi, I changed `Fprint` to `Fprintf` is when using sttr in command line, the output have a '$' in the end of line.
eg:
image

Comment (zhangjieke) [reply cmd/generate.go:169]: and now it likes this
image

**Pull request closed without merging**